src/containers/addNew.js

```diff
@@ -7,42 +7,45 @@ import classNames from 'classnames';
 import './form.scss';
 import { addNewBeer } from '../actions';
 
+const emptyColors = 'rgb(255,239,212)';
+const hoverColors = 'rgb(173, 21, 21)';
+const dimensions = '30px';
+
 class AddNew extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      touched: {
-        beerName: false,
-        rating: false,
-        desc: false,
-      },
-      beerName: '',
-      rating: 1,
-      desc: '',
-      file: '',
-      isDisabled: true,
-    };
-  }
+  state = {
+    touched: {
+      beerName: false,
+      rating: false,
+      desc: false,
+    },
+    beerName: '',
+    rating: 1,
+    desc: '',
+    isDisabled: true,
+  };
 
   changeTheRating = newRating => {
     this.setState({ rating: newRating });
   };
 
+  disableButton = (beerName, desc) => {
+    if (beerName !== '' && desc !== '') {
+      this.setState({
+        isDisabled: false,
+      });
+    } else
+      this.setState({
+        isDisabled: true,
+      });
+  };
```

Comment:
This function says "disable button", but it takes two atributes nad sometimes disables and sometimes enables it.
Thats wrong. Name should match what functions does